frappe/patches/v13_0/remove_twilio_settings.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2015, Frappe Technologies Pvt. Ltd. and Contributors
+# MIT License. See license.txt
+
+import frappe
+
+
+def twilio_settings_doctype_in_integrations() -> bool:
+	"""Check Twilio Settings doctype exists in integrations module or not.
+	"""
+	return frappe.db.exists("DocType", {'name': 'Twilio Settings', 'module': 'Integrations'})
+
+def execute():
```

Comment:
Minor nitpick: Shouldn't `execute` be the first method in the file? And utility methods at the end of the file?